kernel/sched.c

```diff
@@ -1280,12 +1280,21 @@ static inline void z_vrfy_k_thread_priority_set(k_tid_t thread, int prio)
 void z_impl_k_thread_deadline_set(k_tid_t tid, int deadline)
 {
 	struct k_thread *thread = tid;
+	int32_t newdl = k_cycle_get_32() + deadline;
```

Comment:
I immediately read this as "noodle" 😅 Just a comment, not suggesting any changes.

Reply:
Knödel